packages/react/src/NavList/NavList.test.tsx

```diff
@@ -246,20 +246,16 @@ describe('NavList.Item with NavList.SubNav', () => {
 
   it('does not have active styles if SubNav contains the current item and is open', () => {
     const {container, queryByRole} = render(
-      <ThemeProvider>
-        <SSRProvider>
-          <NavList>
-            <NavList.Item>
-              Item
-              <NavList.SubNav>
-                <NavList.Item href="#" aria-current="page">
-                  Sub Item
-                </NavList.Item>
-              </NavList.SubNav>
+      <NavList>
```

Comment:
I see the `<ThemeProvider>` was removed here and was just curious if we removed it intentionally? 🤔

Reply:
@TylerJDev It was not intentionally ;)

Reply:
@TylerJDev It seems like the npx changeset only allows me to create a Major release... I do not believe this is a major release :)